### PR TITLE
Release 1.21.0 

### PR DESCRIPTION
**Pull request by sugarraysam**
# py-mtcli

## Description

Release 1.21.0 :: adds support for OPM 1.19.5 when building bundles.

### CODE REVIEW

**Review by sandhyadalavi — APPROVED** [review]
lgtm

File: managedtenants/bundles/addon_bundles.py [hunk 1]
```diff
@@ -140,7 +140,7 @@ def _validate_config(self, config):
         except jsonschema.exceptions.ValidationError as e:
             raise AddonBundlesError(f"schema validation error for {self}: {e}")
 
-    @lru_cache()
+    @lru_cache(maxsize=128)
```
Comment (Ajpantuso) [inline managedtenants/bundles/addon_bundles.py:143]: nit: can this fix be separated to it's own commit in this PR? Just a little cleaner git history IMO.

Comment (sugarraysam) [reply managedtenants/bundles/addon_bundles.py:143]: sure thing, fixed

**Review by Ajpantuso — APPROVED** [review]
LGTM
